privacyidea/static/css/signin.css

```diff
@@ -103,3 +103,26 @@ body {
 .deactivated {
     text-align: center;
 }
+
+/*
+.ng-invalid.radio label,
+.ng-invalid.checkbox label,
+.ng-invalid.radio-inline label,
+.ng-invalid.checkbox-inline label {
+  color: #a94442;
+}
+*/
+
+input.ng-invalid:not(#username):not(#password),
+select.ng-invalid {
+    background: rgba(169, 68, 66, 0.05);
+  border-color: #a94442;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+          box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+}
+input.ng-invalid:focus:not(#username):not(#password),
+select.ng-invalid:focus:not(#username):not(#password){
```

Comment:
Well, strictly speaking, I think we do not need the ``:not...`` stuff for ``select``, but I think this works :)